address/opcode.go

```diff
@@ -0,0 +1,567 @@
+package address
+
```

Comment:
Aren't all these opcodes already importable from [btcd/txscript](https://github.com/btcsuite/btcd/blob/master/txscript/opcode.go)? Why do we need to redefine them here?

Reply:
Yes, opcodeArray on the other hand is not exported and it is needed in ParseScript method which is also unexported method in btcd